src/components/WorkersList/WorkersList.tsx

```diff
@@ -0,0 +1,73 @@
+import { WorkerItem } from "../WorkerItem/WorkerItem";
+import { Worker } from "../../moudels/Worker";
+import { ReactNode, useState } from "react";
+
+export const WorkersList = () => {
+  const [workerItems, setWorkerItems] = useState<ReactNode[]>([]);
```

Comment:
I don't want you to save components to the state of the application.
Try saving something else and make the components dumb enough to get that object and create whatever it needs from the information